style.css

```diff
@@ -0,0 +1,290 @@
+    margin-left: 20px;
+}
+
+.hero-video {
+    position: relative;
+    height: 100vh;
+    width: 100%;
+    overflow: hidden;
+    margin-bottom: 30px;
+}
+
+.hero-video video {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    min-width: 100%;
+    min-height: 100%;
+    width: auto;
+    height: auto;
+    z-index: -1;
+    transform: translate(-50%, -50%);
+    object-fit: cover;
+}
+
+/*navigation menu */
+
+.navbar {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+    font-family: Lato, Arial, sans-serif;
+    word-spacing: 10px;
+    padding: 10px 0;
+    position: absolute;
+    top: 0;
+    left: 50%;
+    transform: translateX(-50%);
+    width: auto;
+}
+
+/*links */
+
+a {
+    color: black;
+    text-decoration: none;
+}
+
+
+/*text section */
+
+h2 {
+    font-family: Lato, Arial, sans-serif;
+    text-transform: uppercase;
+    text-align: center;
+    font-size: 1.25rem;
+    margin: 10px 0;
+}
+
+p {
+    font-family: Baskerville, Times, serif;
+    font-size: 1.1 rem;
+    text-align: center;
+    margin: 80px;
+    margin-top: 20px;
+    margin-bottom: 40px;
+}
+
+/*heading inside form */
+
+h3 {
+    font-family: Lato, Arial, sans-serif;
+    text-transform: uppercase;
+    font-size: 1rem;
+    margin: 10px 0;
+}
+
+/*form */
+
+form {
+    font-family: Baskerville, Times, serif;
+    border: 1px solid grey;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+    padding: 10px;
+    margin: 10px auto;
+    margin-bottom: 0px;
+    max-width: 90%;
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+/* placeholders in inputs */
+
+::placeholder {
+    text-align: left;
+    font-family: Baskerville, Times, serif;
+    color: black;
+}
+
+/*input fields in form */
+
+input {
+    width: 100%;
+    max-width: 280px;
+    font-size: 0.875rem;
+    padding: 8px;
+    border: none;
+    border-bottom: 1px solid black;
+    background-color: whitesmoke;
+    margin: 5px 0;
+}
+
+/*date of birth input */
+
+.date-label {
+    width: 100%;
+    max-width: 200px;
+    font-family: Baskerville, Times, serif;
+    margin: 5px 0;
+}
+
+.date-label input[type="date"] {
+    width: 100%;
+    font-size: 0.875rem;
+    border: none;
+    border-bottom: 1px solid black;
+    background-color: whitesmoke;
+    margin-top: 5px;
+}
+
+/*terms and conditions */
+
+.checkbox {
+    display: inline-flex;
+    align-items: center;
+    font-family: Baskerville, Times, serif;
+    white-space: nowrap;
+    margin: 5px 0;
+}
+
+/*sibmit button */
+
+button {
+    background-color: black;
+    color: white;
+    border: 1px solid white;
+    width: 100%;
+    max-width: 150px;
+    font-size: 0.875rem;
+    font-weight: bold;
+    padding: 8px;
+    margin: 10px 0;
+    cursor: pointer;
+}
+
+button:hover {
+    background-color: white;
+    color: black;
+    border: 1px solid black;
+}
+
+
+
+/* Responsive design */
```

Comment:
everything looks great and responsive except for the box containing Sign up. Could look great if it stays as a smaller rectangle on wider screen. It looks great on a phone though